task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksListItem.vue

```diff
@@ -220,6 +220,7 @@
               this.$root.$emit('show-alert', {type: 'success',message: this.$t('alert.success.task.unCompleted')});
             }
             this.$root.$emit('update-cart', task);
+            this.$root.$emit('update-task-completed', task);
```

Comment:
Should we send the event when task is marked as incompleted ? what happens if the filter  show completed tasks is enabled ?